src/nix/repl.cc

```diff
@@ -900,10 +905,21 @@ struct CmdRepl : StoreCommand, MixEvalArgs
 
     void run(ref<Store> store) override
     {
-        evalSettings.pureEval = false;
-        auto repl = std::make_unique<NixRepl>(searchPath, openStore());
+        auto state = getEvalState();
+        auto repl = std::make_unique<NixRepl>(searchPath, openStore(),state
```

Comment:
It’d be nicer to split out the `getValue` function to make things easier to read imho